jest.config.js

```diff
@@ -1,6 +1,30 @@
-const neutrino = require('neutrino');
+process.env.REACT_APP_NODE_ENV =
+  process.env.REACT_APP_NODE_ENVNODE_ENV || 'test';
+process.env.REACT_APP_DASHBOARD = process.env.REACT_APP_DASHBOARD || 'awfy';
 
-process.env.NODE_ENV = process.env.NODE_ENV || 'test';
-process.env.DASHBOARD = process.env.DASHBOARD || 'awfy';
+module.exports = {
+  // The root of your source code, typically `/src`
+  roots: ['<rootDir>/src'],
```

Comment:
This seems incorrect, because of that jest can't find the test files that are in `<rootDir>/_tests_`

see https://jestjs.io/docs/configuration#roots-arraystring